.changeset/khaki-jokes-smoke.md

```diff
@@ -0,0 +1,2 @@
+---
+---
```

Comment:
I think this could be merged even without changelog since the command is till hidden

Reply:
Isn't there a CI step that checks for one?

Reply:
I don't think it's required 🤔 
Maybe there's a warning but it shouldn't block merging afaik